api/user.go

```diff
@@ -437,7 +437,7 @@ func getProfiles(c *Context, w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	etag := app.GetUsersEtag()
+	etag := app.GetUsersEtag() + params["offset"] + params["limit"]
```

Comment:
Maybe just throw a `.` between them so that offset 1 and limit 11 doesn't get confused for offset  11 and limit 1